src/dev/docker/Dockerfile

```diff
@@ -149,7 +149,7 @@ COPY kernels /usr/local/share/jupyter/kernels
 COPY kernels/ir/Rprofile.site /usr/local/sage/local/lib/R/etc/Rprofile.site
 
 # Build a UTF-8 locale, so that tmux works -- see https://unix.stackexchange.com/questions/277909/updated-my-arch-linux-server-and-now-i-get-tmux-need-utf-8-locale-lc-ctype-bu
-RUN apt -y install locales && echo "en_US.UTF-8 UTF-8" > /etc/locale.gen && locale-gen
+RUN echo "en_US.UTF-8 UTF-8" > /etc/locale.gen && locale-gen
```

Comment:
PS: the change you quoted above is only removing the 'apt -y install locales' because locales is already installed at the first step of the docker file.

Also, I think using "apt" is discouraged for scripts, should use apt-get instead which has a stable UI.

One could also considered changing the locale to something like "C.UTF-8" which feels more "neutral" although I guess it would be equivalent in terms of translations (but e.g. sorting and stuff is old-school, that's what 'ENV LC_ALL C.UTF-8' is doing).

At least installing and using a locale with the UTF-8 part means unicode characters work in the terminal. For me that's indispensable, while translations are important but a bit less so.